cfg/rgl_base.cfg

```diff
@@ -42,21 +42,20 @@ mp_tournament_stopwatch "1"                      // sets stopwatch mode if appli
 mp_tournament_readymode "0"                      // traditional whole team ready up
 mp_teams_unbalance_limit "0"                     // disables team balance checking
 
-sv_mincmdrate "66"                               // sets the min value for cl_cmdrate to 66 ticks/sec
-sv_maxcmdrate "66"                               // sets the max value for cl_cmdrate to 66 ticks/sec
-sv_minrate "80000"                               // sets min acceptable client rate to 80000 packets/sec, tf2's default rate
+sv_mincmdrate "30"                               // sets the min value for clients to send usercmds to the server to 30 per second
+sv_maxcmdrate "66"                               // sets the max value for clients to send usercmds to the server to 66 per second
+sv_minrate "65535"                               // sets min acceptable client rate to 65535 bytes/sec, a sane value below tf2's default rate but not by a huge margin
 sv_maxrate "0"                                   // sets max acceptable client rate to "unlimited", tho anything higher than 1048576 is ignored as it is tf2's "max" rate.
-sv_minupdaterate "66"                            // sets min rate of requestable packets from server to client to 66 per sec
-sv_maxupdaterate "66"                            // sets max rate of requestable packets from server to client to 66 per sec
+sv_minupdaterate "30"                            // sets min rate of packets from server to client to 30 per sec
+sv_maxupdaterate "66"                            // sets max rate of packets from server to client to 66 per sec
 sv_client_min_interp_ratio "1"                   // clamps min client interp settings
 sv_client_max_interp_ratio "2"                   // clamps max client interp settings
-sv_client_cmdrate_difference "0"                 // clamps client cmdrate
+sv_client_cmdrate_difference "0"                 // clamps client cmdrate to the same as client's updaterate, these should never not match each other unless you want huge client prediction issues
 sv_client_predict "1"                            // forces cl_predict 1, essentially
-
-// sv_clockcorrection_msecs "15"                 // attempts to better synchronize client's ticks closer to server's ticks. currently commented while i test possible hitreg issues
-                                                 // ^ more info: https://github.com/VSES/SourceEngine2007/blob/master/se2007/game/server/player.cpp#L3036
-// sm_cvar sv_maxusrcmdprocessticks_holdaim 2    // prevents pSilentAim from working (hopefully)
-// sm_cvar tf_flamethrower_vecrand 0             // fixes random flamethrower velocity
```

Comment:
this is a league decision. don't comment it, just readd it if rgl ever wants to.